test/conformance/kuttl/mutate/clusterpolicy/standard/existing/basic-delete/01-manifests.yaml

```diff
@@ -51,8 +51,8 @@ spec:
         - apiVersion: v1
           kind: Secret
           name: test-secret-2
-          namespace: "{{ request.object.metadata.namespace }}"
+          namespace: staging-2
         patchStrategicMerge:
           metadata:
             labels:
-              foo: "{{ request.object.metadata.name }}"
\ No newline at end of file
+              foo: dictionary-2
```

Comment:
Logged https://github.com/kyverno/kyverno/issues/6189 for admission variables in DELETE requests.